ratis-logservice/src/main/java/org/apache/ratis/logservice/server/BaseServer.java

```diff
@@ -39,6 +45,23 @@ public ServerOpts getServerOpts() {
     return opts;
   }
 
+  /**
+   * Sets common Ratis server properties for both the log and metadata state machines.
+   */
+  void setRaftProperties(RaftProperties properties) {
+    // Set the ports for the server
+    GrpcConfigKeys.Server.setPort(properties, opts.getPort());
+    NettyConfigKeys.Server.setPort(properties, opts.getPort());
+
+    // Ozone sets the leader election timeout (min) to 1second.
+    TimeDuration leaderElectionTimeoutMin = TimeDuration.valueOf(1, TimeUnit.SECONDS);
+    RaftServerConfigKeys.Rpc.setTimeoutMin(properties, leaderElectionTimeoutMin);
+    TimeDuration leaderElectionMaxTimeout = TimeDuration.valueOf(
+        leaderElectionTimeoutMin.toLong(TimeUnit.MILLISECONDS) + 200,
+        TimeUnit.MILLISECONDS);
+    RaftServerConfigKeys.Rpc.setTimeoutMax(properties, leaderElectionMaxTimeout);
```

Comment:
Same, why are we hardcoding these values?

Reply:
This was a copy-paste out of Ozone. I don't have a good understand of why they chose these.

Was going for just calling out things we _want_ to set in Ratis, but leaving the actual configuration to the work Vlad is going to get to.